pfdl_scheduler/petri_net/drawer.py

```diff
@@ -37,7 +36,8 @@
 
 def draw_graph(graph, attr):
     """Set attributes for drawing the net."""
-    attr["nodesep"] = NODE_SEP_VALUE
+    attr["margin"] = 15
+    attr["newrank"] = "true"
```

Comment:
string'ed "true"?

Reply:
This is necessary because it is parsed by the Snakes library into a boolean for the dot engine (double checked in the corresponding dot file: in here, there is no string'ed true)